src/psyclone/gocean1p0.py

```diff
@@ -57,18 +57,19 @@
 from psyclone.parse.utils import ParseError
 from psyclone.parse.algorithm import Arg
 from psyclone.psyir.nodes import Loop, Literal, Schedule, Node, \
-    KernelSchedule, StructureReference
+    KernelSchedule, StructureReference, BinaryOperation, Reference, \
+    Return, IfBlock
 from psyclone.psyGen import PSy, Invokes, Invoke, InvokeSchedule, \
     CodedKern, Arguments, Argument, KernelArgument, args_filter, \
     AccessType, ACCEnterDataDirective, HaloExchange
 from psyclone.errors import GenerationError, InternalError
 from psyclone.psyir.symbols import SymbolTable, ScalarType, ArrayType, \
     INTEGER_TYPE, DataSymbol, ArgumentInterface, RoutineSymbol, Symbol, \
-    ContainerSymbol, DeferredType, TypeSymbol, GlobalInterface
+    ContainerSymbol, DeferredType, TypeSymbol, GlobalInterface, \
```

Comment:
pylint complains that Symbol and GlobalInterface are not used.

Reply:
Deleted